src/modules/plotNFT.ts

```diff
@@ -148,6 +149,7 @@ export function pwJoinPool(
         poolUrl,
         relativeLockHeight,
         targetPuzzlehash,
+        fee,
```

Comment:
Same question regarding `fee?` being an optional and being passed into a function that may(?) not handle it being undefined.

Reply:
Yeah, I will add default value as well